tp/tp.py

```diff
@@ -146,24 +159,18 @@ def guess_target(self):
         previous = self.target
 
         # How many instances we should keep running
-        elapsed_time = time.time() - self.last_change
-        if elapsed_time > self.cool_down_threshold:
-            candidate = round(self.weight_factor * self.tapping_group.desired_capacity)
+        candidate = round(self.weight_factor * self.tapping_group.desired_capacity)
 
-            # Never less than one
-            if candidate < 1:
-                candidate = 1
+        # Never less than one
+        if candidate < 1:
+            candidate = 1
 
-            max_candidates = self.conf.get("max_candidates", 6)
-            candidate = min(candidate, max_candidates)
-            self.logger.debug("Current candidate for target instances: %s", str(candidate))
+        max_candidates = self.conf.get("max_candidates", 6)
+        candidate = min(candidate, max_candidates)
```

Comment:
Since we don't want `candidate` to be less than one, what do you think about doing something like this:
```
candidate = max(1, min(candidate, self.conf.get("max_candidates", 6)))
```
If so, remove the previous `if`.